widget_driver/lib/src/drivable_widget.dart

```diff
@@ -98,6 +122,17 @@ class _DriverWidgetState<Driver extends WidgetDriver> extends State<DrivableWidg
     // There was no mocked driver. Then just return the hard coded TestDriver.
     return widget.driverProvider.buildTestDriver();
   }
+
+  void _markDriverToBeRebuilt() {
+    _shouldRebuildDriver = true;
+  }
+
+  bool get _driverExists => _driver != null;
+
+  void _disposeDriver() {
+    _shouldRebuildDriver = false;
+    _driver!.dispose();
```

Comment:
Make it possible to call the method even if no driver exists without crashing
```suggestion
    _driver?.dispose();
```